runtime/filetype.vim

```diff
@@ -1410,6 +1414,10 @@ au BufNewFile,BufRead *.mix,*.mixal		setf mix
 " MMIX or VMS makefile
 au BufNewFile,BufRead *.mms			call dist#ft#FTmms()
 
+" msmtp
+" TODO: just for generic syntax for now
+au BufNewFile,BufRead .msmtprc			setf conf
```

Comment:
This now  moved to ~/.config/msmtp/config. See also https://github.com/marlam/msmtp/pull/148/files

Reply:
it actually was a symblic  ~/.msmtprc